ui/src/components/style/FormStyle.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+
+// This is a temporary styling wrapper before we merge this styling in core-ui component
+export const CheckboxWrapper = styled.div`
+  .sc-checkbox {
+    :before {
+      width: 13px;
+      height: 13px;
+    }
+
+    i {
+      top: 3px;
+      left: 2px;
+      font-size: 12px;
+    }
+
+    input[type='checkbox'] {
+      margin: 0px;
+    }
+
+    span {
+      vertical-align: baseline;
+    }
+  }
+`;
+
+export const ToggleWrapper = styled.div`
```

Comment:
Would make sense to create a PR in core-ui for the `CheckboxWrapper` and `ToggleWrapper`